test/widgets/lightbox_test.dart

```diff
@@ -197,6 +204,153 @@ class FakeVideoPlayerPlatform extends Fake
 void main() {
   TestZulipBinding.ensureInitialized();
 
+  group('LightboxHero', () {
+    late PerAccountStore store;
+    late FakeApiConnection connection;
+
+    final channel = eg.stream();
+    final message = eg.streamMessage(stream: channel,
+      contentMarkdown: ContentExample.imageSingle.html, topic: 'test topic');
```

Comment:
nit: order elements logically: so stream, topic, then content, because the stream and topic give the more general context where the content appears